src/context.ts

```diff
@@ -31,7 +30,7 @@ type ContextOutput = {
   type: ContextType;
   vkHash: Field;
   nonce: Field;
-  verifierIdentity: PublicKey | Bytes;
+  verifierIdentity: Bytes;
```

Comment:
hey, `Bytes` + keccak hash would make this hard to use in a zkapp

can you instead use the following type in the zkapp case:
```ts
type AccountId = { address: PublicKey; tokenId: Field }
```

it can be serialized as follows:
```ts
 let { address, tokenId } = accountId;
 let json = { address: address.toBase58(), tokenId: TokenId.toBase58(tokenId) };
 ```
 where `TokenId` is imported from o1js

Reply:
oh, network is also a good idea to add!